webbpsf/wfirst.py

```diff
@@ -362,15 +405,102 @@ def _validate_config(self, **kwargs):
             # If the user has set the pupil to a custom value, let them worry about the
             # correct shape it should have
             pass
+
+    def user_pupil_mask_override(self, set_pupil_mask_on):
+        if set_pupil_mask_on is not None:
```

Comment:
The `if set_pupil_mask_on is not None:` seems unnecessary. If you take that line out, then if called with a None, it would just go to the TypeError exception stating that the parameter must be boolean. That seems like good behavior. 

As written now, if passed a None this function would silently do nothing and raise no error. Is that the desired behavior?

Reply:
I didn't think this one through